javascript/modules/client/src/internal/client-core.ts

```diff
@@ -123,4 +154,20 @@ export abstract class ClientCore implements IClientCore {
       params.creatorChoice ?? VoteOption.NONE,
     ];
   }
+
+  protected calculateGasFeeEstimation(
+    gasLimitEstimationFromCall: Promise<BigNumber>
+  ): Promise<IGasFeeEstimation> {
+    return Promise.all([this.maxFeePerGas, gasLimitEstimationFromCall]).then(
+      data => {
+        return {
+          avg: data[0]
+            .mul(data[1])
+            .div(this.feeEstimationReducer)
```

Comment:
Careful with this. If the reduction factor is `0.625` and we **divide** by that value, we are doing the inverse of a reduction

Reply:
Not exactly. Because we are dealing with `BigNumber` (and therefore we cannot multiply neither divide by floats), the return of `this.feeEstimationReducer` is the factor multiplied by 100 and this is the value we use to divide. And that's why on the next line we multiply by 100, to keep the factor correctly calculated.